docs/src/man/syntax.md

```diff
@@ -418,8 +418,8 @@ g(x) = cos(x) - x
 and then we plot `f` over the interval from ``-π`` to ``π``
 
 ```@example 1
-x = linspace(-π, π)
-plot(x, f(x), color = "red")
+x = LinRange(-π, π, 16)
```

Comment:
Would the canonical choice here just be `range`? LinRange is slightly weird leftover struct, which might sometimes be desirable for performance.

Reply:
```suggestion
x = range(-π, π; length=50)
```